setup.py

```diff
@@ -578,6 +579,13 @@ def pxd(name):
         'pyxfile': '_libs/tslibs/parsing',
         'pxdfiles': ['_libs/src/util',
                      '_libs/src/khash']},
+    '_libs.tslibs.resolution': {
+        'pyxfile': '_libs/tslibs/resolution',
+        'pxdfiles': ['_libs/src/util',
+                     '_libs/src/khash',
+                     '_libs/tslibs/frequencies',
```

Comment:
add tseries_depends here rather than np_dateime

Reply:
I'll do this, but will reiterate my objection to mixing 'pxfiles' and 'depends' keys, as well as to over-specifying.

Reply:
pls do it my way; things broke the other way